examples/camera/main.py

```diff
@@ -1,11 +1,13 @@
-import os
-import kivy3
 from kivy.app import App
 from kivy.clock import Clock
 from kivy.core.window import Window
-from kivy3 import Scene, Renderer, PerspectiveCamera, Vector3
-from kivy3.loaders import OBJMTLLoader
 from kivy.uix.floatlayout import FloatLayout
+from kivy3 import PerspectiveCamera
+from kivy3 import Renderer
+from kivy3 import Scene
+from kivy3 import Vector3
```

Comment:
Just curious why splitting the imports? I'm more used to actually grouping them right? Also `isort` and `black` would do that by default

Reply:
IMHO one import per line makes the code easier to read. Also unused imports are easier to find if static code analysis marks such lines.

Reply:
Don't mind much (if this stays on kivy3 :smile:), but it's the first time I see it. It feels slightly overkill to me, but this is definitely opinionated, unless PEP8 says so.
Since by default I don't have strong opinion on style I try to stick to standard and linters and then I become opinionated as I try to follow them for consistency :smile: 
So I would definitely suggest you to give `isort` and/or `black` a try. I mean for your own knowledge, not that I would be enforcing it in kivy :wink: